composer.json

```diff
@@ -34,7 +35,7 @@
         "sonata-project/user-bundle": "For user timezone detection"
     },
     "conflict": {
-        "sonata-project/user-bundle": "<2.0 || >=4.0"
+        "sonata-project/user-bundle": "<2.0 || >=5.0"
```

Comment:
In this case, we are using the conflict section is like the require section, but for optional deps. It is built by negating a sensible version constraint, and we never use `>`, always `~` or `^`, so I'd rather keep the upper limit.